src/Microsoft.PowerShell.CoreCLR.Eventing/DotNetCode/Eventing/EventDescriptor.cs

```diff
@@ -11,25 +11,25 @@ namespace System.Diagnostics.Eventing
     public struct EventDescriptor
     {
         [FieldOffset(0)]
-        private ushort _id;
+        private readonly ushort _id;
```

Comment:
The struct could be readonly too.

Reply:
`readonly` structs were a new feature in C# 7.2 , so there are likely many structs we can mark `readonly`.

See also: https://devblogs.microsoft.com/premier-developer/the-in-modifier-and-the-readonly-structs-in-c/

Reply:
I don't think there are many, but you can open a new tracking issue.

Reply:
Issue opened: #13915

Reply:
@iSazonov I have PR for readonly structs: #13919